.github/workflows/ci.yml

```diff
@@ -0,0 +1,27 @@
+name: CI
+on: push
+
+jobs:
+  package:
```

Comment:
I'd say we don't need to package in CI, the "Run test is enough"

Reply:
Oh, right. I will just compile and run tests here.